llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -812,6 +812,8 @@ bool VPlanTransforms::adjustFixedOrderRecurrences(VPlan &Plan,
     if (auto *FOR = dyn_cast<VPFirstOrderRecurrencePHIRecipe>(&R))
       RecurrencePhis.push_back(FOR);
 
+  VPBuilder MiddleBuilder(
```

Comment:
Worth renaming `Builder` here to, say, `LoopBuilder`?

Reply:
Updated, thanks!